planetarium/NineChronicles
# Update common UI.

### Description

1. Update common UI. (`Mail`, `Quest`, `Inventory`)

### How to test

1. Check if inventory tab's sprite is changed properly.
2. Check `UI_Mail`, `UI_Quest` and `Inventory` prefab and check if the buttons are toggle.

### Required Reviewers

@planetarium/9c-dev @Namyujeong
